erpnext/selling/report/address_and_contacts/test_address_and_contacts.py

```
@@ -0,0 +1,122 @@
+		filters = frappe._dict(party_type="Customer")
+
+		columns, data = execute(filters)
+
+		self.assertTrue(len(columns) > 0, "Report should return columns")
+		self.assertTrue(len(data) > 0, "Report should return at least one row")
+
+	def test_address_and_contacts_columns_include_designation(self):
+		"""Test that the designation column is present in the report output."""
+		filters = frappe._dict(party_type="Customer")
+
+		columns, data = execute(filters)
```

Ruff (0.15.12)

[warning] Unpacked variable `data` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)